### PR TITLE
Csv config fixes

### PR DESCRIPTION
**Pull request by zvonkok**
This PR fixes several bugs for NFD.
- Updated CSV with correct version
- Updated CSV and added contact email (`operator-courier` was not complaining)
- Added new state to configure Namespace
- Added namespace to CR for confiruation where to install the operand

### CODE REVIEW

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -147,7 +150,11 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: cluster-nfd-operator
+                - name: NODE_FEATURE_DISCOVERY_IMAGE
+                  value: quay.io/openshift/origin-node-feature-discovery:latest
+#                  value: "quay.io/zvonkok/node-feature-discovery:v4.2"
```
Comment (sjug) [inline manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: Drop comment(s) (incl below)

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go [hunk 2]
```diff
@@ -34,6 +34,42 @@ func (s ResourceStatus) String() string {
 	return names[s]
 }
 
+func Namespace(n NFD) (ResourceStatus, error) {
+
```
Comment (sjug) [inline pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Extra `\n` here, I see that's your code style in the following functions.

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go [hunk 3]
```diff
@@ -34,6 +34,42 @@ func (s ResourceStatus) String() string {
...
+	state := n.idx
```
Comment (sjug) [inline pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Your variable names are not ideal, especially the NFD struct: rec, ins, idx. I don't know what any of these are at a glance, see: [variable names](https://github.com/golang/go/wiki/CodeReviewComments#variable-names).

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go [hunk 4]
```diff
@@ -34,6 +34,42 @@ func (s ResourceStatus) String() string {
...
+	obj := n.resources[state].Namespace
+
+	if len(n.ins.Spec.OperandNamespace) != 0 {
+		obj.SetName(n.ins.Spec.OperandNamespace)
+	}
+
+	found := &corev1.Namespace{}
+
```
Comment (sjug) [inline pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Sometimes this extra `\n` is here, sometimes it's not. Lets keep it consistent.

Comment (zvonkok) [reply pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Got it fixed.

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go (hunk 2 above)
Comment (zvonkok) [reply pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Adds a little more whitespace to the functions.

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml (hunk 1 above)
Comment (zvonkok) [reply manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: Done.

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go (hunk 3 above)
Comment (zvonkok) [reply pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Lets keep that in mind for the next release.

**Comment by sjug** [comment]
/lgtm

Pending squash

File: manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml (hunk 1 above)
Comment (zvonkok) [reply manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml]: Done

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go (hunk 2 above)
Comment (zvonkok) [reply pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Done, thanks.

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go (hunk 3 above)
Comment (zvonkok) [reply pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Lets keep this in mind for the next release.

File: pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go (hunk 4 above)
Comment (zvonkok) [reply pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go]: Got it, thanks.

**Pull request merged**